ui/src/app/utils/access-utils.tsx

```diff
@@ -218,7 +224,15 @@ export const useIsUserDisabled = () => {
 };
 
 export const getAccessModuleStatusByName = (profile: Profile, moduleName: AccessModule): AccessModuleStatus => {
-  return profile.accessModules.modules.find(a => a.moduleName === moduleName);
+  let moduleStatus = profile.accessModules.modules.find(a => a.moduleName === moduleName);
+  // Set today's date to byPass for eraCommon in case Registered tier, and era is not Required
```

Comment:
I think there's a difference between "bypassed" and "the module doesn't exist at all".  The way I'm reading the mocks, we want the latter.

Reply:
We would need to bypass this module otherwise it will not display the "Next" button properly for training. As per the current code we will see incomplete training as following
<img width="1233" alt="Screen Shot 2021-10-13 at 11 34 27 AM" src="https://user-images.githubusercontent.com/34481816/137166245-6aa87112-a435-4e20-a431-f474504636e0.png">